Makefile

```diff
@@ -169,9 +169,10 @@ run:
 .PHONY: docker-component # Not intended to be used directly
 docker-component: check-component
 	GOOS=linux GOARCH=amd64 $(MAKE) $(COMPONENT)
```

Comment:
Can't we reuse binaries from `otelcontribcol-all-sys` ? It avoids to declare manualy each platform to build.